src/d3d9/d3d9_spec_constants.h

```diff
@@ -28,6 +28,7 @@ namespace dxvk {
     SpecPixelShaderBools,   // 16 bools                       | Bits: 16
 
     SpecFetch4,             // 1 bit for 16 PS samplers       | Bits: 16
+    SpecDrefClamp,          // 1 bit for 20 VS + PS samplers  | Bits: 20
```

Comment:
Veeeery minor nit, but I'd move this before Fetch4 because its more common.

(Same for bit definition below.)